### DISCUSSION TITLE
External plugins (.vst/.vst3/.so)

### DISCUSSION THREAD

**Discussion by davidhaley** [wwmm/easyeffects#2370] [Ideas]
Hello,

It would be awesome if we could load an external plugin in easyeffects. I use [yabridge](https://github.com/robbert-vdh/yabridge) to wrap .vst/.vst3 in a .so that can be loaded in Linux.

I've used Carla to host vsts (e.g. Sonarworks), and it works; however, I find Carla very unstable.

Is this feature on the radar?

Thank you,
David

**Comment by wwmm** [comment]
There is a discussion about this on #1433. It would be a nice thing to have but I have to be honest. As things are right now it is really hard to implement because being a general purpose host like Ardour or Carla was never the goal.
